frontend/libs/studio-components/src/components/StudioContentMenu/StudioMenuTab/StudioMenuTabContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import type { ReactElement } from 'react';
+import classes from './StudioMenuTabContainer.module.css';
+import { moveFocus } from '../utils/dom-utils';
+import type { StudioMenuTabAsLinkType, StudioMenuTabType } from '../types/StudioMenuTabType';
+import { StudioMenuTabAsLink } from './StudioMenuTabAsLink';
+import { StudioMenuTab } from './StudioMenuTab';
+
+type StudioMenuTabProps<TabId extends string> = {
+  contentTab: StudioMenuTabType<TabId>;
+  isTabSelected: boolean;
+  onClick: (tabId: TabId) => void;
+};
+
+export function StudioMenuTabContainer<TabId extends string>({
+  contentTab,
+  isTabSelected,
+  onClick,
+}: StudioMenuTabProps<TabId>): ReactElement {
+  const handleKeyDown = (event: React.KeyboardEvent<any>) => {
```

Comment:
Can we use another type than `any` here? 😄

Reply:
ups forgot to update that one - yes of course :D